src/test/java/uk/gov/justice/digital/hmpps/riskprofiler/services/PollPrisonersServiceTest.java

```diff
@@ -75,27 +75,23 @@ public void testChangeSoc() {
         service.pollPrisoner(OFFENDER_1);
 
         //todo verify arguments
-        var payload = RiskProfileChange.builder()
+        RiskProfileChange.builder()
                 .newProfile(
                         ProfileMessagePayload.builder()
                                 .soc(SocProfile.socBuilder().nomsId(OFFENDER_1).transferToSecurity(true).build())
                                 .extremism(EXTREMISM_1)
                                 .escape(ESCAPE_1)
                                 .violence(VIOLENCE_1)
-                                .executeDateTime(LocalDateTime.now())
-                                .offenderNo(OFFENDER_1)
-                                .status(Status.NEW)
                                 .build())
                 .oldProfile(
                         ProfileMessagePayload.builder()
                                 .soc(SOC_1)
                                 .extremism(EXTREMISM_1)
                                 .escape(ESCAPE_1)
                                 .violence(VIOLENCE_1)
-                                .executeDateTime(LocalDateTime.now())
-                                .offenderNo(OFFENDER_1)
-                                .status(Status.NEW)
-                                .build()).build();
+                                .build())
+                .executeDateTime(LocalDateTime.now())
+                .offenderNo(OFFENDER_1).build();
 
```

Comment:
should this use the RiskProfileChange defined above?

Reply:
it's used now the verify has been written